sheetScript.gs

```diff
@@ -479,6 +480,8 @@ function deletePlaylistItems(playlistId, deleteBeforeTimestamp) {
         publishedBefore: deleteBeforeTimestamp, // this compares the timestamp when the video was added to playlist
         pageToken: nextPageToken});
         
+      results.items.forEach(x => allVideos.push(x));
```

Comment:
Can this be moved into the for loop, specifically after the removal on line 490 so that we don't remove any videos that were already removed?

Reply:
yes, thanks, fixed in commit 3a4bb61